jsiSIE/jsiSIE/SieDocument.cs

```diff
@@ -196,13 +196,15 @@ public void ReadDocument(Stream stream)
             _fileName = "(stream)";
             ReadStreamAux(stream);
         }
-        
```

Comment:
I need to point out that the standard says codepage 437 but I like the intent of this change anyway. 
This will break the checksum calculation since it also uses 437.
I could probably add codepage as a property of SieDocument and throw an exception when if the codepage is not single byte and the document contains KSUMMA.